java/src/main/java/com/azure/graph/bulk/impl/GraphBulkExecutor.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.graph.bulk.impl;
+
+import com.azure.cosmos.models.CosmosBulkOperationResponse;
+import reactor.core.publisher.Flux;
+
+/**
+ * Interface for classes capable of writing Edges and Vertices to a data store
+ *
+ * @param <V> Defines the class for the Vertices
+ * @param <E> Defines the class for the Edges
+ */
+public interface GraphBulkExecutor<V, E> {
```

Comment:
In theory there should be not restrictions on the number of distinct POJO classes passed in right?

eg. There can be a `Employee`, `Company` vertex types passed in same Iterable.

Reply:
Correct, that's pretty much an leaky abstraction. I'll make sure I can plug that hole.